pkgs/top-level/all-packages.nix

```diff
@@ -13807,6 +13807,8 @@ with pkgs;
     boost = boost159;
   };
 
+  roon = callPackage ../servers/roon { };
```

Comment:
Should this be called `roon-server` to be in line with the nixos module, or is it fine to leave it at `roon`?

Reply:
I'll update it to be `roon-server`, because "roon" would be the UI instead of the server.